runtime/fs.js

```diff
@@ -65,7 +65,7 @@ function make_path_is_absolute() {
     var isUnc = Boolean(device && device.charAt(1) !== ":");
 
     // UNC paths are always absolute
-    if (Boolean(result[2] || isUnc)) {
+    if (result[2] || isUnc) {
```

Comment:
The `Boolean` above might be unnecessary as well.

Reply:
If you change it, the value will be `boolean | ""`

Reply:
Can you be specific ?

Reply:
In this case, an empty string should return an empty string without further evaluation beyond Logical AND.

This should work:

```js
var isUnc = device.length > 0 && device.charAt(1) !== ":";
```

Reply:
Ok, can you make that change ?

Reply:
@hhugo Done!